design-proposals/configurable-features.md

```diff
@@ -0,0 +1,159 @@
+# Overview
+
+With the introduction
+of [KubeVirt Feature Lifecycle](https://github.com/kubevirt/community/blob/main/design-proposals/feature-lifecycle.md)
+policy, features reaching General Availability (GA) need to drop their use of feature gates. This applies also to
+configurable features that we may want to disable.
+
+## Motivation
+
+Users may want certain features to be in a given state, for example to make the best use out of given
+resources or for compliance reasons features may expose sensitive information from the host to the virtual machines (VM)
+or add additional containers to the launcher pod, which are not required by the user. For instance, the downward metrics
+feature is a good example of why some clusters may want to have it enabled or disabled.
+The downward metrics feature exposes some metrics about the host node where the VM is running to the guest. This
```

Comment:
nit - can you break this into a separate paragraph for easier review?

Reply:
Done.